Content.Client/GameObjects/EntitySystems/VerbSystem.cs

```diff
@@ -196,6 +198,43 @@ private void _fillEntityPopup(VerbSystemMessages.VerbsResponseMessage msg)
 
                 buttons.Add(button);
             }
+            //Get global verbs. Visible for all entities regardless of their components.
+            foreach (var globalVerb in VerbUtility.GetGlobalVerbs(Assembly.GetExecutingAssembly()))
+            {
+                if (globalVerb.RequireInteractionRange)
```

Comment:
I can't help but feel a lot of this code can be somehow deduplicated.

Reply:
I was thinking the same. It looks bad, and there's only slight differences between the two loops. I'll see what I can do with it.

Reply:
@PJB3005 Cleaned up both client & server versions of `VerbSystem`. Code is much smaller now. Shouldn't be much duplicate code left.